tests/model_registry/model_catalog/huggingface/test_huggingface_model_validation.py

```
@@ -1,23 +1,66 @@
+      syncInterval: '2m'
+""",
+                "microsoft/phi-2",
+                "microsoft/phi-2",
+                id="test_hf_last_synced_custom",
+            ),
+        ],
+        indirect=["updated_catalog_config_map_scope_function", "initial_last_synced_values"],
+    )
+    def test_huggingface_last_synced_custom(
+        self: Self,
+        updated_catalog_config_map_scope_function: Generator[ConfigMap, str, str],
```

Ruff (0.14.13)

Unused method argument: `updated_catalog_config_map_scope_function`

(ARG002)